source/exe/main_common.cc

```diff
@@ -21,80 +22,112 @@
 
 #include "ares.h"
 
+#include <pthread.h>
```

Comment:
nit: is this needed?

Reply:
no; was using getpid() for debugging.  removed.